FusionIIIT/applications/complaint_system/models.py

```diff
@@ -73,8 +73,8 @@ class StudentComplain(models.Model):
     complaint_type = models.CharField(choices=Constants.COMPLAINT_TYPE,
                                       max_length=20, default='internet')
     location = models.CharField(max_length=20, choices=Constants.AREA)
-    specific_location = models.CharField(max_length=50, blank=True)
-    details = models.CharField(max_length=100)
+    specific_location = models.CharField(max_length=100, blank=True)
+    details = models.CharField(max_length=1000)

```

Comment:
The max length for a Charfiled is 255 only . Why do we need to change this, can you explain?

Reply:
Because, we are getting details and 100 character in detail means only one line, but complain could be more than 4-5 lines so for that reason I have taken this length and if we have more than 4-5 lines of complain detail than only scrollbar is feasible.